app/src/main/java/com/example/superheroapi/data/AppContainer.kt

```diff
@@ -33,15 +33,16 @@ interface AppContainer {
  *
  * Variables are initialized lazily and the same instance is shared across the whole app.
  */
-class DefaultAppContainer : AppContainer {
-    private val baseUrl = "https://android-kotlin-fun-mars-server.appspot.com/"
 
+const val BASE_URL = "https://superheroapi.com/api/"
```

Comment:
Corrected the URL and changed it to a const val, so the compiler can set it at compile time.